### ISSUE TITLE
[FR] Add command for "Chat: Show Chats - Open in Editor"

### ISSUE THREAD

**Issue by wenfangdu** [microsoft/vscode-copilot-release#2133]
Currently, after using `workbench.action.chat.history`, we can only click the "Open in Editor" button to open the chat in the editor area, which is inefficient for keyboard users, would be nice to have a command for that, so we can assign a keyboard shortcut to it (such as `enter`) instead of using mouse to click it.
![Image](https://github.com/user-attachments/assets/1a20ca63-6d36-49ba-bd88-ae755c12dfd5)

**Comment by roblourens** [comment]
You can tab to it. Maybe a keybinding would be nice, like cmd+enter

**Comment by wenfangdu** [comment]
> You can tab to it.

@roblourens No, you can't, I just tried, even if it does, it's very inconvenient compared to a shortcut.

> Maybe a keybinding would be nice, like cmd+enter

Yes, please add a command for it, it only takes little work, but can hugely improve many devs' workflow.

**Comment by roblourens** [comment]
Yes you can, note that you have to use the arrow keys to navigate within the toolbar once you have tabbed to the toolbar.

**Comment by wenfangdu** [comment]
@roblourens That's too complicated and counter-intuitive for a simple action, poor accessibility for keyboard users, please add a command for it, thank you.

**Comment by wenfangdu** [comment]
@roblourens If it gets closed, I'll recreate this issue until it gets solved.
